src/api/env-variables.ts

```diff
@@ -1,23 +1,25 @@
 export interface EnvVariables {
-  [key: string]: string | number | boolean;
+  [key: string]: { value: string | number | boolean, secure: boolean };
 }
 
-export function set(envs: EnvVariables, key: string, value: string | number | boolean): void {
-  envs[key] = value;
+export function set(
+  envs: EnvVariables, key: string, value: string | number | boolean, secure = false
+): void {
+  envs[key] = { value: value, secure: secure };
 }
 
 export function unset(envs: EnvVariables, key: string): void {
-  envs[key] = null;
+  envs[key] = { value: null, secure: false };
```

Comment:
please set to `null` instead of `false` when unsetting.

Reply:
actually to correctly unset env variable it can be:
```ts
envs[key] = null;
```
or even:
```ts
delete envs[key];
```
what do you think?

Reply:
yes, the last one is probably the way to go...

Reply:
done